README.md

```diff
@@ -53,48 +111,50 @@ Since the executor runs inside its own docker container it will try to reach its
 To run the logstash framework as a Marathon app use the following app template (save e.g. as logstash.json):
 Update the JAVA_OPTS attribute with your Zookeeper servers.
 
- ```
- {
-   "id": "/logstash",
-   "cpus": 1,
-   "mem": 1024.0,
-   "instances": 1,
-   "container": {
-     "type": "DOCKER",
-     "docker": {
-       "image": "mesos/logstash-scheduler:0.0.6",
-       "network": "HOST"
-     }
-   },
-   "env": {
-     "JAVA_OPTS": "-Dmesos.logstash.framework.name=logstash_framework -Dmesos.zk=zk://<zkserver:port>,<zkserver:port>/mesos"
-   }
- }
- ```
- 
- Note: See <a href="#fw_configuration">Framework options</a> for all available options.
+```json
```

Comment:
This needs a little wax on-wax off. @sadovnikov Could you post your latest Marathon config here?